core/src/main/java/org/elasticsearch/search/internal/InternalSearchHit.java

```diff
@@ -808,7 +810,9 @@ public void writeTo(StreamOutput out, InternalSearchHits.StreamContext context)
             out.writeVInt(innerHits.size());
             for (Map.Entry<String, InternalSearchHits> entry : innerHits.entrySet()) {
                 out.writeString(entry.getKey());
-                entry.getValue().writeTo(out, InternalSearchHits.streamContext().streamShardTarget(InternalSearchHits.StreamContext.ShardTargetType.NO_STREAM));
+                ShardTargetType previous = InternalSearchHits.streamContext().streamShardTarget();
```

Comment:
Is there a better name for this variable? I'm not really sure what previous is here? Could this not just be `shardTarget`?